src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -980,32 +981,35 @@ namespace AirLoopHVACDOAS {
 
     void AirLoopDOAS::SizingAirLoopDOAS(EnergyPlusData &state)
     {
-        Real64 sizingMassFlow = 0;
+        Real64 sizingVolumeFlow = 0;
 
         for (int AirLoop = 1; AirLoop <= this->NumOfAirLoops; AirLoop++) {
             int AirLoopNum = this->m_AirLoopNum[AirLoop - 1];
             this->m_OACtrlNum.push_back(state.dataAirLoop->AirLoopControlInfo(AirLoopNum).OACtrlNum);
 
             if (this->m_OACtrlNum[AirLoop - 1] > 0) {
-                sizingMassFlow += state.dataMixedAir->OAController(this->m_OACtrlNum[AirLoop - 1]).MaxOA;
+                sizingVolumeFlow +=
+                    state.dataMixedAir->OAController(this->m_OACtrlNum[AirLoop - 1]).MaxOA; // this is a volume flow rate not a mass flow rate
             }
         }
-        this->SizingMassFlow = sizingMassFlow;
+        this->SizingMassFlow = sizingVolumeFlow * state.dataEnvrn->StdRhoAir;
+
+        BaseSizer::reportSizerOutput(state, "AirLoopHVAC:DedicatedOutdoorAirSystem", this->Name, "Design Volume Flow Rate [m3/s]", sizingVolumeFlow);
```

Comment:
Most (all?) other equipment reports sizing as "Design Size Air Flow Rate [m3/s]" or "Design Size Maximum Air Flow Rate". This should probably use similar language. @rraustad ?

Reply:
Usually it's "Design Size" or User-Specified"  pre-pended on the front end of the field names which are not uniform. But this object doesn't have a field for air flow so it's a sum of this field. I have no preference for this object since it's not autosized or hard-sized but I guess it could match what the air loop reports since that's what gets summed.

    AirLoopHVAC,
       N1, \field Design Supply Air Flow Rate

The air loop doesn't follow this naming convention either.

    Component Sizing Information, AirLoopHVAC, VAV SYS 1, Design Supply Air Flow Rate [m3/s], 1.08612

Unitary System looks like this:

 Component Sizing Information, AirLoopHVAC:UnitarySystem, DX COOLING COIL, Design Size Cooling Supply Air Flow Rate [m3/s], 1.34859
 Component Sizing Information, AirLoopHVAC:UnitarySystem, HEATPUMP DX COIL 1, Design Size Heating Supply Air Flow Rate [m3/s], 1.34859

TUs:

    Component Sizing Information, AirTerminal:SingleDuct:VAV:Reheat, SPACE1-1 VAV REHEAT, Design Size Maximum Air Flow Rate [m3/s], 0.20149